lib/master-logger.js

```diff
@@ -23,7 +22,6 @@ function _configureLogger(name) {
   var logger = new (winston.Logger)({
     transports: [
       new winston.transports.File(config.logger.file),
-      new winston.transports.Slack(config.logger.Slack),
```

Comment:
We can still use Slack for error logging! The Node 12 upgrades use the API (and a different package) - see the errors that got posted recently to the #aggie-deployments channel 😄 

Also see #326 for how we can making debugging errors in development easier (without having to send them to a Slack channel)

Reply:
Great! I did see the errors logged on Slack. We will use your updated master-logger when we merge.